bin/templates/scripts/cordova/lib/run.js

```diff
@@ -200,12 +201,22 @@ function startSim (appPath, target) {
     return iossimLaunch(appPath, 'com.apple.CoreSimulator.SimDeviceType.' + target, logPath, '--exit');
 }
 
-function iossimLaunch (app_path, devicetypeid, log, exit) {
+function iossimLaunch (appPath, devicetypeid, log, exit) {
     var f = path.resolve(path.dirname(require.resolve('ios-sim')), 'bin', 'ios-sim');
-    var proc = cp.spawn(f, ['launch', app_path, '--devicetypeid', devicetypeid, '--log', log, exit]);
-    proc.stdout.on('data', (data) => {
-        console.log(data.toString());
-    });
+    var params = ['launch', appPath, '--devicetypeid', devicetypeid, '--log', log, exit];
+
+    return superspawn.spawn(f, params, { cwd: projectPath, printCommand: true, stdio: 'pipe' })
```

Comment:
If you use `stdio: inherit` here, you should be able to drop the whole `progress` block.

Reply:
The initial idea was to be able to handle the error case different than the normal output that ios-sim "logs". Will that work without all this as well?

Reply:
Ah right. You do the prefixing. Then nevermind my comment. You can drop the stdio option though, since pipe is the default.